resources/windows/wix/sequences.wxs.erb

```diff
@@ -14,7 +14,9 @@
         CMDLINE_INSTALLDIR
       </Custom>
       <Custom Action='SetIniPropertyValues' After='AppSearch' />
-      <Custom Action='GetLocalisedAccountNames' After='AppSearch' />
+      <Custom Action='GetLocalisedAccountNames' After='AppSearch'>
```

Comment:
Is there no other way of configuring this action?  `SetIniPropertyValues` runs in unattended mode no problem, so I'm not sure this makes sense to me?